winex.js

```diff
@@ -360,7 +362,7 @@ factory = function(winstonLogger, classMeta, opts) {
           callback = typeof metaOrCb === "function" ? metaOrCb : null;
         }
         meta = this._makeMeta(level, meta);
-        return winstonLogger[meta.level].apply(Log, [msg, meta, callback]);
+        return winstonLogger[meta.level].apply(winstonLogger, [msg, meta, callback]);
```

Comment:
without this change, the winstonLogger was not bound to the Log object correctly. I did not see any negative side effects

Reply:
`:yolo emoji:` 😄